lib/modules/@apostrophecms/areas/src/apos/components/ApostropheAddWidgetMenu.vue

```diff
@@ -79,6 +79,6 @@ export default {
 <style>
 .apos-area-add-widget-menu {
   width: 100px;
-  border: 2px solid lightblue;
+  border: 2px solid var(--a-brand-blue);
```

Comment:
Is there a reason these have actual color names in their names, rather than some sort of semantic name?

Reply:
@stuartromanek ?